src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -510,6 +509,26 @@ class TypeSpecTest {
         |""".trimMargin())
   }
 
+  @Test fun sealedClassesMayDefineAbstractMembers() {
+    val sealedClass = TypeSpec.classBuilder("Sealed")
+        .addModifiers(KModifier.SEALED)
+        .addProperty(PropertySpec.builder("name", String::class)
+                             .addModifiers(ABSTRACT).build())
```

Comment:
The indentation here is broken again..

Reply:
I meant to ask about that.  Is there a `kotlinpoet` configuration I can import?  that formatting is done by the (iirc) default formatter in IDEA.

Reply:
Yes, https://github.com/square/java-code-styles. It's been updated to format Kotlin too.